Update 00-login.md

marc-wittwer opened the pull request:
When I ran `npx react-native run-android` the `${account.namespace}` variable could not be resolved.

This comment would have helped me.

<!---
Pull Requests for Quickstart Guides can still be submitted here, but most other documentation content is no longer hosted on GitHub and therefore no longer open-sourced. If you are an Auth0 employee trying to make a change, please [submit a ticket](https://auth0team.atlassian.net/servicedesk/customer/portal/9). Thank you!
--->

joshcanhelp committed:
fix: redirect individual social connections to marketplace

joshcanhelp committed:
fix: redirect individual social connections to marketplace

joshcanhelp committed:
fix: redirect individual social connections to marketplace

joshcanhelp committed:
fix: redirect individual social connections to marketplace

joshcanhelp committed:
fix: add salesforce community

joshcanhelp committed:
fix: redirect order for more clear review changeset

joshcanhelp committed:
fix: remove instagram redirect

joshcanhelp committed:
fix: remove instagram redirect

joshcanhelp committed:
fix: add sso redirects

joshcanhelp committed:
fix: remove SSO sidebar

rkhoriander committed:
Add log streaming redirects

abbaspour committed:
nginx+ quickstart

avanscoy committed:
Update redirects.js

Moving page out of Tokens section to Config under API

avanscoy committed:
Update redirects.js

marc-wittwer committed:
Update 00-login.md

When I ran `npx react-native run-android` the `${account.namespace}` variable could not be resolved.

This comment would have helped me.

avanscoy committed:
Update redirects.js

avanscoy committed:
Update redirects.js

rkhoriander committed:
Add index page

avanscoy committed:
Update redirects.js

Widcket committed:
Fix snippet in Touch ID / Face ID QS (#9815)

Commit:
[Quickstart] Ionic 5, Capacitor, Auth0 React quickstart (#9763)

* Add initial content + login steps

Also refactored some common content from the React quickstart into an
include.

* Add logout steps

* Add user profile section

* Further edits and tweaks

* Update download.md for Ionic React

* Updates based on review feedback

* Fixed some wording around localstorage

* Various tweaks to add clarity after running through the tutorial again

* Update repo and requirements for Ionic React

Commit:
[SDK-2535] Ionic 5, Capacitor & Auth0 Angular quickstart (#9784)

* Add initial content + login steps

Also refactored some common content from the React quickstart into an
include.

* Add logout steps

* Add user profile section

* Further edits and tweaks

* Update download.md for Ionic React

* Updates based on review feedback

* Fixed some wording around localstorage

* Various tweaks to add clarity after running through the tutorial again

* Update repo and requirements for Ionic React

* Add Ionic + Angular quickstart and refactor out some common includes for the React one

* Tweaks to the intro and requirements

* Fix typo

* Add origin for Android

* Apply suggestions from code review

Co-authored-by: Frederik Prijck <frederik.prijck@auth0.com>

* Fix code samples for Angular based on review

* Add redirect from current ionic4 quickstart to ionic-angular

* Remove old ionic4 quickstart

* Modify repo for Ionic Angular sample

* Update tags and framework

* Fix include in Ionic React QS

* Reconfigure initialization snippets to remove need for local storage + RTs by default

* Tweak wording in storage note

* Add note about SSO on iOS with capacitor browser plugin

Co-authored-by: Frederik Prijck <frederik.prijck@auth0.com>

Commit:
Fix closing backtick in NGINX quickstart (#9816)

tamigoodall committed:
Hide Redundant Rules Best Practice docs

Hide Redundant Rules Best Practice docs

tamigoodall committed:
Support sidebar updates

Support sidebar updates

tamigoodall committed:
Update sidebar.yml

Change title of Auth0 Overview to How You Can Use Auth0

tamigoodall committed:
Update sidebar.yml

Added Protocols to Authorization

renoirtech committed:
Updating User model namespace to match Laravel 8

abbaspour committed:
incorporate review from f5 marketing team. Nginx+ => NGINX Plus. changed QS type from community to Auth0 owned.

abbaspour committed:
Accept-Encoding "gzip"  for jwks.json as well

Commit:
Fix GitHub integration for Ionic quickstart (#9822)

* fix: use correct branch for ionic samples

* Remove old Ionic folder

evansims committed:
docs: Update panel referencing new beta SDK, quickstarts, etc.

Commit:
Update 01-login.md

Commit:
Update 01-authorization.md

tamigoodall committed:
Arch Scenarios - sidebar fix

Arch Scenarios - sidebar fix

tamigoodall committed:
Update sidebar.yml

Title match fix

tamigoodall committed:
Update sidebar.yml

Typo fix

tamigoodall committed:
Add section to Connections Enterprise SAML doc

From DOCS-1184

tamigoodall committed:
Added more detail

Added more detail

rkhoriander committed:
Update saml.md

stevehobbsdev commented:
@marc-wittwer The `${account.namespace}` variable is actually something specific to our docs app. If you're signed in when you view the docs, that variable should have been replaced with your Auth0 domain. If you're not signed in, it should have been replaced with the words `YOUR_DOMAIN`.

Looking at the quickstart, I can see that the variable is not replaced properly - let me look into that.

I think the note is useful but the variable needs to remain as-is.

jimmyjames committed:
Simplify Spring Security Dependencies (#9827)

abbaspour committed:
address issue from qs-feedback. missing semicolon.

rkhoriander committed:
Fix broken links with /docs

rkhoriander committed:
Update glossary.json

rkhoriander committed:
Update glossary.json

rkhoriander committed:
Update glossary.json

rkhoriander committed:
Update glossary.json

lbalmaceda committed:
Fix 404 in quickstart link

rkhoriander committed:
Update glossary.json

erjohnson committed:
Update amazon connection logo (#9831)

joshuarichards001 committed:
missing end quote (#9832)

CustomCodeIT committed:
Update 02-calling-an-api.md (#9639)

When running through the documentation on a brand new react project, I noticed the user metadata wasn't ever displaying because the useEffect was firing a single time before the user gets populated. By adding "getAccessTokenSilently" and "user?.sub" to the useEffect dependency array, I was able to get this code working reliably.

You can see in the console logs from the catch block that the first fire fails as the user has not yet been defined. When the user is updated, useEffect fires again which causes the metadata to populate properly.

rkhoriander committed:
Remove redirect from main sso integration page.

rkhoriander committed:
Turn off goodreads

pclavier92 committed:
fix: update node version

rkhoriander committed:
Add Criipto redirects

rkhoriander committed:
Add back single sign-on integrations main page

rkhoriander committed:
Replace backticks

rkhoriander committed:
Replace backticks

misterpantz committed:
Update redirects.js

add support redirect

misterpantz committed:
Update redirects.js

misterpantz committed:
Update redirects.js

misterpantz committed:
Update sidebar.yml

added support breakup changes

frederikprijck committed:
Add Reference to use Webview2 with WPF and Winforms (#9835)

frederikprijck committed:
[SDK-2511] Add QuickStart for ASP.NET MVC Beta SDK (#9726)

* Add QuickStart for ASP.NET MVC Beta SDK

* Rework beta quickstart

* Update quickstart with new API

frederikprijck committed:
Update GitHub path for ASP.NET MVC Beta Quickstart (#9837)

tamigoodall committed:
Update redirects.js to fix broken links in new nav from Contentful (#9839)

* Update redirects.js

* more fixes

libraries and integrations link corrections

* SAML protocol fixes

SAML protocol fixes

* fix sessions and troubleshoot redirects

fix sessions and troubleshoot redirects

Commit:
Fix auth0Domain replacement

stevehobbsdev commented:
Closing this in favour of #9841, just to be safe as there's something funky going on with the history. Thanks again for the contribution! 🙌🏻
